tests/foreman/ui/test_org.py

```diff
@@ -259,6 +259,25 @@ def test_positive_delete_bz1225588(self, org_name):
                     break
             self.assertIsNone(status)
 
+    def test_manifest_refresh_bz1259248(self):
+        org = entities.Organization().create()
+        sub = entities.Subscription(organization=org)
+        with open(manifests.download_manifest_template(), 'rb') as manifest:
+            upload_manifest(org.id, manifest)
+        try:
+            with Session(self.browser) as session:
+                for _ in range(3):
+                    self.assertIsNotNone(org.download_debug_certificate())
+                    session.nav.go_to_select_org(org.name)
+                    session.nav.go_to_red_hat_subscriptions()
+                    self.subscriptions.refresh()
+                    self.assertIsNone(session.nav.wait_until_element(
+                        common_locators['notif.error'], timeout=5))
+                    self.assertTrue(session.nav.wait_until_element(
+                        common_locators['alert.success'], timeout=180))
+        finally:
+            sub.delete_manifest(data={'organization_id': org.id})
```

Comment:
Manifest must be deleted no matter what result our test case gave as we should not impact other test cases by our failures here